virttest/http_server.py

```diff
@@ -52,7 +54,7 @@ def copyfile_range(self, source_file, output_file, range_begin, range_end):
         output_file.write(buf)
 
     def send_head_range(self, range_begin, range_end):
-        path = self.translate_path(self.path)
+        path = self.translate_path(decode_to_text(self.path))
```

Comment:
this may not place-in to decode_to_text, translate_path() method may be more suitable place.

Reply:
Fixed